var/spack/repos/builtin/packages/py-healpy/package.py

```diff
@@ -25,3 +25,5 @@ class PyHealpy(PythonPackage):
     depends_on("py-six", type=("build", "run"))
     depends_on("cfitsio", type=("build", "run"))
     depends_on("healpix-cxx", type=("build", "run"))
+
+    patch("cfitsio_version_check.patch", when="@1.13:")
```

Comment:
This patch doesn't exist. Can you also add a comment explaining why it's needed?

Reply:
The patch is necessary for the code to compile. The original code made some assumptions when checking the version of a dependency that does not hold true. The patch is not fully fleshed out as it is not clear what the developers actually need to check when compiling the code. Definitely, the current check is inadequate and will fail with newer `cfitsio` versions. The patch effectively removes the check and I rely on spack to determine whether the satisfies the version requirements.

Reply:
Is it the same patch for both healpix-cxx and py-healpy? If so you'll need to copy it to the py-healpy directory and/or modify it.